client/my-sites/domains/domain-management/edit/style.scss

```diff
@@ -38,3 +38,55 @@
 		}
 	}
 }
+
+@mixin domain_status_color( $color ) {
+	border-top: 5px solid $color;
+	> div.domain-status__icon {
+		color: $color;
+		> svg {
+			fill: $color;
+		}
+	}
+}
+
+.domain-types__container {
+	.status-success {
+		@include domain_status_color( #008a20 );
```

Comment:
We should be using CSS variables for these colours. They are all defined here: 

https://github.com/Automattic/wp-calypso/blob/master/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss

and are implemented like this:
`@include domain_status_color(  var( --color-success-50 ) );`